workshop/cleanup/clean_your_k8s_cluster.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo ''
+echo 'You are about to remove EVERYTHING on your Kubernetes cluster, including Istio.'
+echo ''
+echo 'You must put in Y as your answer.'
+read -p "Are you sure? " -n 1 -r
+echo
+if [[ $REPLY =~ ^[Y]$ ]]
+then
+  kubectl delete pods --all
+  kubectl delete svc --all
+  kubectl delete deployments --all
+  kubectl delete configmaps --all
+  kubectl delete pods -n istio-system --all
+  kubectl delete svc -n istio-system --all
+  kubectl delete deployments -n istio-system --all
+  kubectl delete namespace istio-system
+fi
```

Comment:
@jjasghar could we change this to run helm del --purge istio or kubectl delete -f istio.yaml and others?

You can find the exact istio uninstall cmds off istio.io.  Then also remove guestbook.  I would much prefer removing what we asked them to install vs delete all.

Reply:
I think the challenge here is that people might not have the `istio.yaml` is the same location. These commands goes out to the `k8s` cluster and nukes everything on there without needing to know the location of someone installing the istio.yamls.

Reply:
@jjasghar sorry for the slow reply.  If we ask folks to remove everything in their k8s cluster, would their cluster still be functional?  Would these cmds remove some of the Armada pods/services etc in the kube-system namespace?

If the cluster is still functional, I'm ok with this.

Reply:
Yep, tested and verified. It does delete everything but k8s brings everything back up to it's in a "clean" state.

Reply:
awesome, thank you @jjasghar